app/views/govuk_publishing_components/components/_layout_super_navigation.html.erb

```diff
@@ -0,0 +1,129 @@
+<%
+  navigation_links = YAML::load("
```

Comment:
YAML was chosen so it can be edited with less technical knowledge, as it's (relatively) a bit easier and less scary than editing a Ruby hash. It's been moved into the translations / localisation file.

Note - there could be problems (regardless of where this is stored) if this is translated and correctly using `hreflang` and `lang` if the translation for the phrase is available (eg "Government activity") but the page that it links to is not translated. It's unlikely to happen soon, as to be best of my knowledge the header and the footer have never been translated - but just to flag that's definitely not a simple problem.